src/components/NextButton.svelte

```diff
@@ -0,0 +1,9 @@
+<div class="row justify-center mt-8">
```

Comment:
`.mt-8` is not a valid bootstrap class. It goes from 0 to 5 https://getbootstrap.com/docs/5.0/utilities/spacing/#margin-and-padding

Reply:
✅ in c791201